app/src/androidTest/java/com/hedvig/app/feature/insurance/detail/ContractDetailScreen.kt

```diff
@@ -28,6 +29,7 @@ class ContractDetailScreen : Screen<ContractDetailScreen>() {
             itemType(::Peril)
             itemType(::Row)
         })
+        val chevron = KImageView { withId(R.id.chevron) }
```

Comment:
This item is not logically on the same `Screen`, right? That's why I had a `PerilBottomSheetScreen` 🙈